website/blog/authors.yml

```diff
@@ -27,9 +34,73 @@ chi:
   image_url: https://github.com/chilagrow.png
   page: true
 
+dmitry:
+  name: Dmitry Eremenko
+  page: true
+
 elena:
   name: Elena Grahovac
   title: FerretDB Team
   url: https://www.linkedin.com/in/grahovac/
   image_url: https://github.com/rumyantseva.png
   page: true
+
+farkasp:
+  name: Peter Farkas
+  title: Co-Founder, CEO
+  url: https://www.linkedin.com/in/farkasp/
+  image_url: https://github.com/ptrfarkas.png
+  page: true
+
+julian:
+  name: Julian König
+  title: Senior Developer at j&s-soft GmbH
+  image_url: /img/blog/julian.jpeg
+  page: true
+
+maciek:
+  name: Maciek Urbanski
+  title: Senior Platform Engineer at Elotl
+  url: https://www.linkedin.com/in/maciekurbanski/
+  image_url: /img/blog/maciek-urbanski.jpg
+  page: true
+
+marcin:
+  name: Marcin Gwóźdź
+  title: Director of Strategic Alliances at FerretDB
+  url: https://www.linkedin.com/in/marcin-gwóźdź-277abaa9
+  image_url: /img/blog/marcin-gwozdz.jpeg
+  page: true
+
+michael:
+  name: Michael Feygelman
+  title: Senior Project and Community Manager for enmeshed at j&s-soft GmbH
+  url: https://www.linkedin.com/in/%F0%9F%92%A1michael-feygelman%F0%9F%92%A1-1590aa1a4/
```

Comment:
This url wasn't specified before correct?

Reply:
No it wasn't. I have removed it